Content/Console/README.md

```diff
@@ -11,27 +11,30 @@ MacOS/Linux | Windows
 [![Travis Badge](https://travis-ci.org/MyGithubUsername/MyLib.1.svg?branch=master)](https://travis-ci.org/MyGithubUsername/MyLib.1) | [![Build status](https://ci.appveyor.com/api/projects/status/github/MyGithubUsername/MyLib.1?svg=true)](https://ci.appveyor.com/project/MyGithubUsername/MyLib.1)
 [![Build History](https://buildstats.info/travisci/chart/MyGithubUsername/MyLib.1)](https://travis-ci.org/MyGithubUsername/MyLib.1/builds) | [![Build History](https://buildstats.info/appveyor/chart/MyGithubUsername/MyLib.1)](https://ci.appveyor.com/project/MyGithubUsername/MyLib.1)  
 
----
+## Nuget 
 
-### Developing
+Stable | Prerelease
+--- | ---
+[![NuGet Badge](https://buildstats.info/nuget/MyLib.1)](https://www.nuget.org/packages/MyLib.1/) | [![NuGet Badge](https://buildstats.info/nuget/MyLib.1?includePreReleases=true)](https://www.nuget.org/packages/MyLib.1/)
 
-- [VSCode Dev Container](https://code.visualstudio.com/docs/remote/containers)
 
 ---
 
-### Building
+### Developing
 
 Make sure the following **requirements** are installed in your system:
 
-- [dotnet SDK](https://www.microsoft.com/net/download/core) 2.0 or higher
+- [dotnet SDK](https://www.microsoft.com/net/download/core) 3.0 or higher
 - [Mono](http://www.mono-project.com/) if you're on Linux or macOS.
 
-```sh
-> build.cmd // on windows
-$ ./build.sh  // on unix
-```
+or
 
-#### Environment Variables
+- [VSCode Dev Container](https://code.visualstudio.com/docs/remote/containers)
+
+
+---
+
+### Environment Variables
 
 - `CONFIGURATION` will set the [configuration](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-build?tabs=netcore2x#options) of the dotnet commands.  If not set it will default to Release.
```

Comment:
"If not set it will default" -> "If not set, it will default"

Reply:
thanks a lot!